jobs/check_akismet_user.rb

```
@@ -0,0 +1,12 @@
+module Jobs
+  class CheckAkismetUser < Jobs::Base
+
+    def execute(args)
+      return if args[:user_id].blank? || args[:profile_content].blank?
```

Usually we want to raise a `Discourse::InvalidParameter` error instead of silently returning from the job.